plugins/crypto-kms/build.gradle

```diff
@@ -30,6 +30,7 @@ dependencies {
   api "software.amazon.awssdk:aws-core:${versions.aws}"
   api "software.amazon.awssdk:utils:${versions.aws}"
   api "software.amazon.awssdk:auth:${versions.aws}"
+  api "software.amazon.awssdk:identity-spi:${versions.aws}"
```

Comment:
Does there need to be a dependency on https://mvnrepository.com/artifact/software.amazon.awssdk/retries-spi as well?

Reply:
Interesting, crypto-kms didnt show any thirdparty missing classes.
let me check.